### DISCUSSION TITLE
How to set s Fixed width for the indicator.

### DISCUSSION THREAD

**Discussion by demolg** [prateekmedia/netspeedsimplified#18] [Q&A] [answered]
![recording](https://user-images.githubusercontent.com/26198098/114306591-cc0a3300-9b0e-11eb-8d24-d67810218e22.gif)
The changing width makes other icon move. In the fixed width, it should be:
Arrows: left align
Digital and Unit: right align

**Comment by prateekmedia** [comment, marked as answer]
You should checkout min width property in Preferences and set it to highest value, that way it would become fixed width.

This is not done by default cause some users dont like space consumption in extension.

**Reply by demolg** [reply]
Thanks for your suggestion
